### PR TITLE
Tabs/Nav-tabs demo

### LINKED ISSUE #228: Nav-tabs demo

### PR DESCRIPTION
**Pull request by DoRightt**
Signed-off-by: cosmicintruder <littlegreenbagg@googlemail.com>

### CODE REVIEW

File: src/app/docs/navigation/demo-tabs/demo-tabs.component.html [hunk 1]
```diff
@@ -1,146 +1,151 @@
...
+                            [data]="[{value: 0, name: 'milkshake'}, {value: 1, name: 'kombucha'}, {value: 2, name: 'smoothie'}]"></sdk-radio>
+                    </div>
+                </sdk-tab>
+                <sdk-tab label="Disabled Tab" [disabled]="true"></sdk-tab>
+                <sdk-tab label="Super-mega-uber-extra-duper-long-tab">
+                    <div class="tab-wrapper">
+                        <h2 class="demo-tabs__title">Pick color</h2>
+                        <sdk-radio
+                            [data]="[{value: 0, name: 'red'}, {value: 1, name: 'blue'}, {value: 2, name: 'green'}]"></sdk-radio>
+                        <h2 class="demo-tabs__title">Pick fruit</h2>
```
Comment (scDisorder) [inline src/app/docs/navigation/demo-tabs/demo-tabs.component.html]: use `*ngFor="let x of arr"` for example templates, not a hardcoded ones
f.e.
```html
<sdk-tab-group [animation]="true">
  <sdk-tab *ngFor="let t of tabs" [label]="t.label" [icon]="t.icon" [disabled]="t.disabled">
      <!-- content injection via ngTemplate/Ref or something -->
  </sdk-tab>
</sdk-tab-group>
```

File: src/app/docs/navigation/demo-tabs/demo-tabs.component.ts [hunk 2]
```diff
@@ -7,9 +8,200 @@ import { Component, OnDestroy, OnInit } from '@angular/core';
...
+                {
+                    title: 'Tabs',
+                    description: '',
+                    templateRef: this.basicRef,
+                    values: {
+                        html: `<div>
+    <h1>Tabs</h1>
+    <sdk-tab-group [animation]="true">
+        <sdk-tab label="Tab 1" icon="star">
+            <div class="tab-wrapper">
```
Comment (scDisorder) [inline src/app/docs/navigation/demo-tabs/demo-tabs.component.ts]: same, dynamic usage examples needed

**Review by scDisorder — CHANGES_REQUESTED** [review]

File: src/app/docs/navigation/demo-tabs/demo-tabs.component.html (hunk 1 above)
Comment (DoRightt) [reply src/app/docs/navigation/demo-tabs/demo-tabs.component.html]: done

File: src/app/docs/navigation/demo-tabs/demo-tabs.component.ts (hunk 2 above)
Comment (DoRightt) [reply src/app/docs/navigation/demo-tabs/demo-tabs.component.ts]: done

**Review by scDisorder — APPROVED** [review]

**Pull request merged**